src/syft/core/common/group.py

```diff
@@ -32,15 +32,21 @@ def proto2object(proto: VerifyKey_PB) -> VerifyKey:
 )
 
 
-@bind_protobuf
-class VerifyAll(Serializable):
-    def _object2proto(self) -> VerifyAllWrapper_PB:
-        return VerifyAllWrapper_PB(all=Empty_PB())
-
-    @staticmethod
-    def _proto2object(proto: VerifyAllWrapper_PB) -> Type:  # type: ignore
-        return VerifyAll
-
-    @staticmethod
-    def get_protobuf_schema() -> GeneratedProtocolMessageType:
-        return VerifyAllWrapper_PB
+def _create_VERIFYALL() -> Any:
```

Comment:
Why not make it a Singleton just to be sure instead of hiding it in a function?

CC: @madhavajay

Reply:
@madhavajay said that we don't want users call like `isinstance(some_key, VerifyAll)`, but want them to use `some_key is VERIFYALL`.

Reply:
I think we can make this a singleton as well just make sure its not possible to get the type and then create a different copy. So there are two things here:
✅ 1) hide the class from import
☑️ 2) make it a singleton so you end up with if if you try to make another one from its type constructor